### ISSUE TITLE
Broken same-page link

### ISSUE THREAD

**Issue by christinach** [pulibrary/orangelight#3886]
### Steps to recreate
Visit a catalog record page. For example go to [record 99125410673606421](https://catalog.princeton.edu/catalog/99125410673606421) 
Evaluate the page using the WAVE browser extension. In the results you will see that there is an alert for a broken link. 

### URL
https://catalog.princeton.edu/catalog/99125410673606421

### Issues
[2.1.1 keyboard](https://www.w3.org/TR/WCAG22/#keyboard)

### WCAG Conformance level (A, AA, AAA)
A

### User impact
A link to another location within the page is present but does not have a corresponding target. 

### Recommended fix
Update the href with a meaningful link. For example: "skip-link"
`<a href="#skip-link" class="back-to-top" style="display: block;">Top</a>`

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...
